app/konnect/dev-portal/audit-logging/webhook.md

```diff
@@ -0,0 +1,140 @@
+    }'
+    ```
+
+    Be sure to replace the following placeholder values:
+    * `<personal-access-token>`: Your {{site.konnect_short_name}} [personal access token (PAT)](/konnect/api/#authentication).
+    * `endpoint`: The external endpoint that will receive audit log messages. Check your SIEM documentation to find out where to send CEF or JSON data.
+    * `authorization`: The authorization type and credential to pass to your log collection endpoint. 
+    {{site.konnect_short_name}} will send this string in the `Authorization` header of requests to that endpoint. For example, if you are setting up the webhook for Splunk, you could provide a Splunk access token: `"authorization":"Splunk example-token12234352535235"`.
+    * `log_format`: The output format of each log message. Can be `cef` or `json`.
+    * `name`: A unique human-readable name to identify this destination.
+    * `skip_ssl_verification`: (Optional) Set to `true` to skip SSL verification of the host endpoint when delivering payloads. We recommend skipping SSL verification only when using self-signed SSL certificates in a non-production environment as this can subject you to man-in-the-middle and other attacks.
+
+    If the request is successful, you will receive a `200` response code, and a response body containing the audit log destination's configuration details. Be sure to save the audit log destination `id` for the next step. 
+
+1. Create a webhook by sending a PATCH request to the [`/audit-log-webhook`](/konnect/api/portal-management/latest/) endpoint with your configured audit log destination:
+
+    ```sh
+    curl -i -X PATCH https://{region}.api.konghq.com/v2/portals/{portalId}/audit-log-webhook \
+     --header "Content-Type: application/json" \
+     --header "Authorization: Bearer <personal-access-token>" \
+     --data '{
+         "audit_log_destination_id": "05atf3f2-9d07-4e46-8115-c58ca594d00e",
+         "enabled": true
+     }'
+    ```
+
+    Be sure to replace the following placeholder values:
+    * `{region}.api.konghq.com`: The region your Dev Portal is located in. Can be `us`, `au`, or `eu`.
+    * `<personal-access-token>`: Your {{site.konnect_short_name}} [personal access token (PAT)](/konnect/api/#authentication).
+    * `{portalId}`: The ID of the Dev Portal with your webhook.
+    * `audit_log_destination_id`: The ID of the audit log destination that you want to use.
+
+    You can't customize the events that {{site.konnect_short_name}} sends to the logs.
+
+    If the request is successful, you will receive a `200` response code, and a response body containing the webhook's configuration details.
+
+{% endnavtab %}
+{% endnavtabs %}
+
+Your webhook should now start receiving audit logs.
+
+## View audit log webhook status
+
+{% navtabs %}
+{% navtab Konnect UI %}
+
+1. In {% konnect_icon dev-portal %} [**Dev Portal**](https://cloud.konghq.com/portal), click the Dev Portal you want to view the webhook status job for.
```

Comment:
Users can also see the status of their Dev Portal webhook in "Organization -> Audit Log Setup -> Dev Portal -> {dev portal name}"